ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/RoundKudf.java

```diff
@@ -20,11 +20,27 @@
 
 public class RoundKudf implements Kudf {
 
+  public static final String NAME = "ROUND";
+
   @Override
   public Object evaluate(final Object... args) {
-    if (args.length != 1) {
-      throw new KsqlFunctionException("Len udf should have one input argument.");
+    if (args.length != 1 && args.length != 2) {
+      throw new KsqlFunctionException("Round udf should have one or two input arguments.");
+    }
+
+    if (args[0] == null) {
+      return null;
+    }
+    final Double number = (Double) args[0];
+
+    if (args.length == 1) {
```

Comment:
As mentioned previously, I think we should treat the second parameter being `null` the same as there being only 1 parameter, so...

```suggestion
    if (args.length == 1 || args[1] == null) {
```